.github/workflows/test.yml

```diff
@@ -4,6 +4,9 @@ name: Test
 on:
   workflow_dispatch:
```

Comment:
Let's add inputs to workflow dispatch as well, just to be able to run specific eve version testing manually

Reply:
We will find later way to have common `on` section